yorkie/log/log.go

```diff
@@ -18,18 +18,53 @@ package log
 
 import (
 	"os"
+	"sync"
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
-// Logger is the default logger used by Yorkie.
-var Logger *zap.SugaredLogger
+var defaultLogger *zap.SugaredLogger
+var coreLogger zapcore.Core
+var rawLogger *zap.Logger
```

Comment:
It seems rawLogger can only be used in the scope of the initialize() function, what do you think?

Reply:
I removed it :)